### ISSUE TITLE
Menu burger modifications

### ISSUE THREAD

**Issue by Kevils** [starknet-id/app.starknet.id#947]
## Description 📹

Current one :
<img width="916" alt="image" src="https://github.com/user-attachments/assets/60a3f9c4-de89-4765-9385-0c552f22f90e" />

New design :
<img width="885" alt="image" src="https://github.com/user-attachments/assets/2239529e-be25-457b-ba29-248b2706fc2f" />

Figma file : https://www.figma.com/design/S1UKYgWewNqNHZFAaBUilG/%F0%9F%8F%9D%EF%B8%8F-Starknet-ID?node-id=4097-32518&t=wEy6O4uIMpUixad7-1

**Modify the icons with new ones and add a #CDCCCC border to the content wrapper.**

## Proposed Actions 🛠️

Here’s a checklist of actions to follow for resolving this issue:

1. **Fork and Create Branch**:  
   Fork the repository and create a new branch using the issue number:
   ```bash
   git checkout -b fix-[issue-number]
   ```

2. **Implement Changes**:  
	[Insert Code snippet if needed with a mardown todo list]

3. **Run Tests and Commit Changes**:  
   Make sure your changes don't break existing functionality and commit with a clear message:
   ```bash
   git commit -m "Fix: [Short description of the fix]"
   ```

## Required 📋

To keep our workflow smooth, please make sure you follow these guidelines:

- **Assignment**: Don't create a pull request if you weren’t assigned to this issue.
- **Timeframe**: Complete the task within **3 business days**.
- **Closing the Issue**: In your PR description, close the issue by writing `Close #[issue_id]`.
- **Review Process**:
  - Once you've submitted your PR, change the label to **"ready for review"**.
  - If changes are requested, address them and then update the label back to **"ready for review"** once done.
- **Testing**: Test your PR locally before pushing, and verify that tests and build are working after pushing.

Thank you for your contribution 🙏

⚠️ WARNING: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.

**Comment by ryzen-xp** [comment]
Can I tackle this one?

**Comment by Amarjeet325** [comment]
I'd love to give this a go.

**Comment by SudiptaPaul-31** [comment]
Can I work on this issue? @Kevils
I am a fullstack and blockchain developer with experience for more than 1.5 years and an active contributor to starkent 
My profile: https://app.onlydust.com/u/SudiptaPaul-31

ETA-1Day

**Comment by mimisavage** [comment]
Is it okay if I tackle this?

**Comment by davedumto** [comment]
Hi,
My name is David, and I am a frontend developer skilled in JavaScript, TypeScript, React.js, Next.js, and Tailwind CSS. I will update the icons with the new design specified in the Figma file and add a #CDCCCC border to the content wrapper as instructed.
I’ll ensure the changes are implemented seamlessly and tested thoroughly to maintain functionality and responsiveness. ETA: 8 hours

**Comment by Ekene001** [comment]
Could I take a shot at this? I’m a full-stack blockchain developer with over 3 years of experience working with Tailwind, React, TypeScript, and Solidity/Rust.

**Comment by chiscookeke11** [comment]
Can I work on this, please?

**Comment by Stepcu** [comment]
Would love to tackle this!

**Comment by BlessingEmejulu** [comment]
Can I start working on the menu burger modifications

**Comment by emarc99** [comment]
May I be assigned to this?

**Comment by code-stan** [comment]
I am a creative frontend developer, keen on implementing accurate and pixel-perfect design.

**Comment by feranmiba** [comment]
Can I attempt this issue?

**Comment by caxtonacollins** [comment]
May I handle this issue?

**Comment by chiscookeke11** [comment]
Can I take care of this issue?

**Comment by Marchand-Nicolas** [comment]
Hello @BlessingEmejulu everything good ?

**Comment by BlessingEmejulu** [comment]
> Hello [@BlessingEmejulu](https://github.com/BlessingEmejulu) everything good ?

![Image](https://github.com/user-attachments/assets/b7ac6243-2953-4a70-bd99-2fd3d14ec1d8)

Currently, facing an internal error from one of the associated third packages (lottie-react) used... will like to get the error fixed ...

**Comment by Marchand-Nicolas** [comment]
> Currently, facing an internal error from one of the associated third packages (lottie-react) used... will like to get the error fixed ...

Do you get this problem too without any changes made to the original code ?

**Comment by caxtonacollins** [comment]
@BlessingEmejulu, i think i solved this issue by downgrading my node version to 18. You can use nvm to switch between versions.

**Comment by BlessingEmejulu** [comment]
Yeah, that fixed it. I downgraded the node version ...

**Comment by Marchand-Nicolas** [comment]
> [@BlessingEmejulu](https://github.com/BlessingEmejulu), i think i solved this issue by downgrading my node version to 18. You can use nvm to switch between versions.

Yes, really good suggestion. I use nvm too to easily manage that

**Comment by BlessingEmejulu** [comment]
![Image](https://github.com/user-attachments/assets/6018ae33-d8b2-4562-b66f-64b70da45d62)

Fixed

**Comment by Marchand-Nicolas** [comment]
> ![Image](https://github.com/user-attachments/assets/6018ae33-d8b2-4562-b66f-64b70da45d62)
> 
> Fixed

I made a review on your PR, please check it
